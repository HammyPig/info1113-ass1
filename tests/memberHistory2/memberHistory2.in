ADD MEMBER john
member history 199999
ADD COLLECTION books.csv
RENT 100000 111111
mEmber hISTORY 100000
RELINQUISH 100000 111111
mEmber hISTORY 100000