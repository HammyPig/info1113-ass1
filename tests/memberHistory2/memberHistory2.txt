outputs history only when book is relinquished, or states if chosen member is not found