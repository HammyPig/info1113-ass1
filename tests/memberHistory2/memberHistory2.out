user: Success.

user: No such member in system.

user: 17 books successfully added.

user: Success.

user: No rental history for member.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
